db/migrate/20150629202819_create_votes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateVotes < ActiveRecord::Migration
+  def change
+    create_table :votes do |t|
+      t.integer :user_id, null: false
```

Comment:
can a user vote on the same review more than once? if not, we should add a unique index and validate uniqueness